content_schemas/examples/worldwide_office/frontend/worldwide_office.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This example is a bit basic for now, but I figure I'll revisit it with some real data once I've published some offices.